Tests/Unit/SOGoTest.h

```diff
@@ -54,6 +54,8 @@
 
 - (BOOL) run;
 
+NSString *_stringForCharacterAtIndex(NSUInteger index, NSString *str, NSUInteger length);
```

Comment:
Getting a warning otherwise.  If it's only used in this file, I guess I should instead turn that function into a static one?